src/controllers/post.js

```diff
@@ -19,6 +19,22 @@ export const create = async (req, res) => {
   }
 }
 
+export const editPost = async (req, res) => {
+  const { content } = req.body
+  console.log(content)
+  const { id } = req.params
+
+  try {
+    const postToEdit = await Post.fromJson(req.body)
```

Comment:
You pass in `req.body`, but you also store `content` as a variable. Both seems redudant - is that the case?

Reply:
removed the content